config/core/resources/jobsink.yaml

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: jobsinks.sinks.knative.dev
+  labels:
+    knative.dev/crd-install: "true"
+    duck.knative.dev/addressable: "true"
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/name: knative-eventing
+spec:
+  group: sinks.knative.dev
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      subresources:
+        status: { }
+      schema:
+        openAPIV3Schema:
+          description: 'JobSink triggers long-running jobs when an event occur.'
+          type: object
+          properties:
+            spec:
+              description: Spec defines the desired state of the JobSink.
+              type: object
+              properties:
+                  job:
+                    type: object
+                    x-kubernetes-preserve-unknown-fields: true
```

Comment:
Could we maybe provide a bit more info here of what we are expecting? From the CRD only, I would have a hard time knowing what to put in the spec here....

Reply:
I've added description and link to the k8s API reference